dbt_subprojects/dex/models/trades/dex_trades.sql

```diff
@@ -29,6 +29,7 @@
                                         , "sei"
                                         , "sonic"
                                         , "sophon"
+                                        , "tac"
```

Comment:
can we remove this file modification, so that CI can run end-to-end?
dex trades is the blocker. we can come back to add `tac` to blockchain list later.

Reply:
Yes, removed!